Form/Type/TranslationsFormsType.php

```diff
@@ -62,14 +64,13 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     public function buildView(FormView $view, FormInterface $form, array $options)
     {
         $view->vars['default_locale'] = $this->localeProvider->getDefaultLocale();
-        $view->vars['required_locales'] = $options['required_locales'];
-    }   
+        $view->vars['required_locales'] = $this->localeProvider->getRequiredLocales();
```

Comment:
This change the logic if someone want override required locales for specific form